appengine/storage.js

```diff
@@ -72,7 +72,7 @@ BlocklyStorage.link = function() {
  * @param {string} key Key to XML, obtained from href.
  */
 BlocklyStorage.retrieveXml = function(key) {
-  var xml = Blockly.Xml.workspaceToDom(Blockly.getMainWorkspace());
+  Blockly.Xml.workspaceToDom(Blockly.getMainWorkspace());
```

Comment:
Strange, this was fixed months ago.  Your copy of Blockly is confused: this file is ancient, while package.json is right up to date.